C/Coin-OR/SHOT/bundled/patches/0002-cmake.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Unrelated to this pull request, can this patch be upstreamed?

Reply:
Yes, I think so.